packages/kbn-test/src/auth/saml_auth.ts

```diff
@@ -58,7 +58,9 @@ const cleanException = (url: string, ex: any) => {
 const getCookieFromResponseHeaders = (response: AxiosResponse, errorMessage: string) => {
   const setCookieHeader = response?.headers['set-cookie'];
   if (!setCookieHeader) {
-    throw new Error(`Failed to parse 'set-cookie' header`);
+    throw new Error(
+      `${errorMessage}: no 'set-cookie' header, response.data: ${JSON.stringify(response?.data)}`
```

Comment:
@azasypkin do we need to redact something specific here?

Reply:
I believe we should be fine:
* Re-using\abusing the logged SAML response is possible, but its lifetime is very short, so having it in the logs **generated by tests** is fine, IMO.
* The SAML response might include PII (user email, name, address), but we’re doing this for test-only users, right? So it should be fine too.